R/tdata.R

```diff
@@ -1,26 +1,26 @@
-#' Create a `tdata` Object
+#' Create a `tdata` object
 #'
 #' @description `r lifecycle::badge("deprecated")`
-#' Create a new object called `tdata` which contains `data`, a `reactive` list of data.frames
+#'
+#' Create a new object called `tdata` which contains `data`, a `reactive` list of `data.frames`
 #' (or `MultiAssayExperiment`), with attributes:
-#' \itemize{
-#'   \item{`code` (`reactive`) containing code used to generate the data}
-#'   \item{join_keys (`join_keys`) containing the relationships between the data}
-#'   \item{metadata (`named list`) containing any metadata associated with the data frames}
-#' }
+#' - `code` (`reactive`) containing code used to generate the data
+#' - join_keys (`join_keys`) containing the relationships between the data
+#' - metadata (named `list`) containing any metadata associated with the data frames
+#'
 #' @name tdata
-#' @param data A `named list` of `data.frames` (or `MultiAssayExperiment`)
+#' @param data A named `list` of `data.frames` (or `MultiAssayExperiment`)
 #'  which optionally can be `reactive`.
 #'   Inside this object all of these items will be made `reactive`.
 #' @param code A `character` (or `reactive` which evaluates to a `character`) containing
 #'   the code used to generate the data. This should be `reactive` if the code is changing
 #'   during a reactive context (e.g. if filtering changes the code). Inside this
 #'   object `code` will be made reactive
-#' @param join_keys A `teal.data::join_keys` object containing relationships between the
+#' @param join_keys (`teal.data::join_keys`) object containing relationships between the
 #'   datasets.
-#' @param metadata A `named list` each element contains a list of metadata about the named data.frame
+#' @param metadata (named `list`) each element contains a list of metadata about the named `data.frame`
```

Comment:
```suggestion
#' @param metadata (`named list`) each element contains a list of metadata about the named `data.frame`
```

Reply:
Not so sure about this one. To me named `list` looks better.

Reply:
will restore "named `list`" everywhere :|